### DISCUSSION TITLE
CDC Support For Snowflake as Source

### DISCUSSION THREAD

**Discussion by AyushiHariaZN** [airbytehq/airbyte#35949] [Connector Ideas and Features]
I am trying to set up a connection, with source as Snowflake and destination as Postgres. For that I am using Incremental Deduped History as the sync method, since I want the changes in the source to be reflected in incremental manner. Apart from that the source deletes the record repeatedly and I want that too be reflected in the destination. I got to know that can be achieved via CDC (Change Data Capture). But when I set Snowflake as source, I got to know it doesn't support the CDC for Snowflake. Is there any way to get the support for CDC in Snowflake as SOURCE?

**Comment by bradrobertson** [comment]
Ya we're also interested in this very same thing. Snowflake natively supports CDC through [streams](https://docs.snowflake.com/en/user-guide/streams). It'd be great if Airbyte could read these!

[a comment by an automated account is not shown]
